contentcuration/contentcuration/models.py

```diff
@@ -1069,8 +1069,13 @@ def save(self, *args, **kwargs):
                 self.checksum = md5.hexdigest()
             if not self.file_size:
                 self.file_size = self.file_on_disk.size
-            if not self.file_format:
-                self.file_format_id = os.path.splitext(self.file_on_disk.name)[1]
+            if not self.file_format_id:
+                ext = os.path.splitext(self.file_on_disk.name)[1].lstrip('.')
+                if ext in dict(file_formats.choices).keys():
+                    self.file_format_id = ext
+                else:
+                    raise ValueError("Files of type `{}` are not supported.".format(ext))
```

Comment:
This should be handled more gracefully in the future.

Reply:
@kollivier I changed this to look at `file_formats.constants.choices`, as you suggested.  Also wrote a test in le_utils to ensure that this stays consistent with all the `FileFormat` entries.

Reply:
Cool, looks good! :)